### ISSUE TITLE
XFCE Notifications settings window not styled correctly.

### ISSUE THREAD

**Issue by BobbyBabes** [adapta-project/adapta-gtk-theme#287]
**Reporter info**

 ```
 * Distribution - Manjaro Minimal Net Edition (16.08)
 * Gtk+ 3.0 version - 3.22.3
 * Desktop environment - XFCE 4.12
 ```

**Adapta version (if you knew)**

Adapta Gtk+ Theme 3.89.1 just installed from the repo.

```

        Adapta Gtk+ Theme 3.89.1
        ===========================================================

        -----------------------------------------------------------
        | Supported Gtk+ version: 3.22 & 3.20 & 3.18
        -----------------------------------------------------------

         prefix:                 /usr
         datarootdir:            ${prefix}/share
         target directories:     ${prefix}/share/themes/Adapta
                                 ${prefix}/share/themes/Adapta-Nokto
                                 ${prefix}/share/themes/Adapta-Eta
                                 ${prefix}/share/themes/Adapta-Nokto-Eta

         parallel-build:         yes
         parallel:               /usr/bin/parallel

         sassc:                  /usr/bin/sassc
         inkscape:               /usr/bin/inkscape
         glib-compile-resources: /usr/bin/glib-compile-resources

         selection-color:        #00BCD4 (default: #00BCD4)
         2nd-selection-color:    #4DD0E1 (default: #4DD0E1)
         accent-color:           #4DB6AC (default: #4DB6AC)
         suggestion-color:       #009688 (default: #009688)
         destruction-color:      #FF5252 (default: #FF5252)

        -----------------------------------------------------------
        | Default DE support
        -----------------------------------------------------------

         GNOME:                >= 3.22 (or 3.20 or 3.18)
         Budgie-Desktop:       >= 10.2.5

        -----------------------------------------------------------
        | Extra DE support
        -----------------------------------------------------------

         Cinnamon:             no
         GNOME-Flashback:      no
         Unity:                no
         XFce:                 yes
         Mate:                 no
         Openbox:              no

        -----------------------------------------------------------
        | Next Gtk+ support
        -----------------------------------------------------------

         Gtk+ 4.0:             no

        -----------------------------------------------------------
        | Extra support
        -----------------------------------------------------------

         Chrome(ium):          yes
         Plank:                no

        ===========================================================
                             =>> run 'make' and 'sudo make install'
```
**Related Application and/or Shell theming (or name of widget)**
Notifications settings.

**Actual issue**
It looks like this window is forgotten by Adapta Noko :

![forgotten_2016-11-22_01-29-41](https://cloud.githubusercontent.com/assets/22870070/20507060/a863acc4-b058-11e6-93f3-795a9a992ff5.png)

**Steps to reproduce (if you knew)**
Applications >> Settings >> Notifications to display the window.

**Expected behaviour**
Window should have dark styling instead of light.

**Other Note (feature-request, question, etc...)**
None.

**Comment by tista500** [comment]
I've already mentioned about global dark mode: https://github.com/adapta-project/adapta-gtk-theme#variant-matrix

You should learn about how Gtk+ 3.x works with colour variants.

**Comment by tista500** [comment]
It's not an issue of Adapta itself.

Closed.

Regards.

**Comment by BobbyBabes** [comment]
If only your writings would mention why someone would want to enable global dark theme.
I don't want to learn about Global Dark Theme. If it can be switched on or off by an installer by updating a settings file. Uninstalling!

**Comment by BobbyBabes** [comment]
Before going back to KDE, and against better judgement, I attempted your writings.
I installed Gnome Tweak Tool. But it didn't show up in XFCE's Applications menu.
Dragging ``gnome-tweak-tool.desktop`` file from ``/usr/share/applications`` onto the panel at the bottom of the screen created the icon launcher, but Gnome Tweak Tool refused to start. Check the ``OnlyShowIn=`` line in the ``gnome-tweak-tool.desktop`` file. Restricted for what reason?
And, not surprisingly, there is no ``$HOME/.config/gtk-3.0/settings.ini`` file.

Lots of loose ends to ghetto-fix.

![gnome-tweak-tool-not-meant-for-xfce_2016-11-22_10-00-47](https://cloud.githubusercontent.com/assets/22870070/20517290/b55941b8-b09a-11e6-9916-e6ebfb501dbe.png)
